langchain4j/src/main/java/dev/langchain4j/model/azure/AzureOpenAiLanguageModel.java

```diff
@@ -0,0 +1,95 @@
+package dev.langchain4j.model.azure;
+
+import dev.ai4j.openai4j.OpenAiClient;
+import dev.ai4j.openai4j.completion.CompletionRequest;
+import dev.ai4j.openai4j.completion.CompletionResponse;
+import dev.langchain4j.model.Tokenizer;
+import dev.langchain4j.model.language.LanguageModel;
+import dev.langchain4j.model.language.TokenCountEstimator;
+import dev.langchain4j.model.openai.OpenAiChatModel;
+import lombok.Builder;
+
+import java.net.Proxy;
+import java.time.Duration;
+
+import static dev.langchain4j.internal.RetryUtils.withRetry;
+import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
+import static java.time.Duration.ofSeconds;
+
+/**
+ * Represents a connection to the Azure OpenAI LLM with a completion interface.
```

Comment:
Represents a connection to the OpenAI LLM, hosted on Azure (like text-davinci-003).